app/logic/Calendar/Invitation/OutgoingInvitation.ts

```diff
@@ -31,15 +31,19 @@ export class OutgoingInvitation {
 
   async sendInvitationsTo(participants: Collection<Participant>) {
     let sendTo = participants.filterOnce(participant => participant.response != InvitationResponse.Organizer);
-    for (let participant of sendTo) {
-      participant.response ||= InvitationResponse.NoResponseReceived;
+    if (sendTo.hasItems) {
+      for (let participant of sendTo) {
+        participant.response ||= InvitationResponse.NoResponseReceived;
+      }
+      await this.send("REQUEST", sendTo);
     }
```

Comment:
```suggestion
    if (sendTo.isEmpty) {
      return;
    }
    for (let participant of sendTo) {
      participant.response ||= InvitationResponse.NoResponseReceived;
    }
    await this.send("REQUEST", sendTo);
```

Early exit. See also below.